src/hooks/useCurrentMarketId.ts

```diff
@@ -60,6 +66,15 @@ export const useCurrentMarketId = () => {
         setLastViewedMarket(marketId);
         dispatch(setCurrentMarketId(marketId));
         dispatch(closeDialogInTradeBox());
```

Comment:
ummm this works although in the next part it'll just essentially ignore it if it passes the conditions

i wonder if we can do the following:
- if active trade box dialog = close position and market has an open position -> do nothing, 
- else close dialog

1. either update market during onCloseButtonToggle navigate 
2. or on the ClosePositionForm update useEffect so it resets market regardless of currentInput

i think if you run the following the market should change on the form and have the right inputs
```
abacusStateManager.setClosePositionValue({
        value: market,
        field: ClosePositionInputField.market,
      });

      abacusStateManager.setClosePositionValue({
        value: SIZE_PERCENT_OPTIONS[MAX_KEY],
        field: ClosePositionInputField.percent,
      });
```

Reply:
don't want to block this further tho -- can ship this and let's treat this as a follow up!

Reply:
filed follow up [here](https://linear.app/dydx/issue/CT-746/follow-up-code-cleanup)